cmd/oci-runtime-tool/generate.go

```diff
@@ -157,8 +157,8 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 	if context.IsSet("label") {
 		annotations := context.StringSlice("label")
 		for _, s := range annotations {
-			pair := strings.Split(s, "=")
-			if len(pair) != 2 {
+			pair := strings.SplitN(s, "=", 2)
+			if len(pair) != 2 || pair[0] == "" {
```

Comment:
Changing from `Split` → `SplitN` makes sense (allowing `=` to occur in the value).  But I'm not clear on this empty-string comparison here.  My understanding is that the spec allows empty-string keys for annotations, and that matches your f2c88e27 commit message.  But this line will lead to “incorrectly specified…” errors when the argument starts with an equals sign.  The `Split` → `SplitN` change may be all we need.